csvcubed/csvcubed/cli/inspect/inspect_sparql_queries/select_codelist_cols_by_dataset_url.sparql

```diff
@@ -0,0 +1,24 @@
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX csvw: <http://www.w3.org/ns/csvw#>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+
+
+SELECT DISTINCT ?columnPropertyUrl ?columnValueUrl ?columnTitle
+WHERE {   
+    BIND (?table_url as ?tableUrlStr).
```

Comment:
Can you put this bind down the bottom by the filter? I think it'd make more sense there.